index.js

```diff
@@ -90,15 +96,23 @@ async function fetchImageUrl(url) {
   try {
     const response = await axios(url);
     const $ = cheerio.load(response.data);
-    const singleImage = $('.page-mainimage').find('img').attr('src');
-    const doubleImage = $(
-      '.swiper-articleimage li.swiper-slide[data-hash="slide1"]',
-    )
-      .find('img.swiper-lazy')
-      .attr('src');
-    const imageUrl = singleImage || doubleImage;
-
-    return `${BASE_URL}${imageUrl}`;
+    const singleImageElement = $('.page-mainimage').find('img');
+    const multipleImageElement = $('.swiper-wrapper').find('img');
+
+    const element = singleImageElement.length
```

Comment:
You could do 
```javascript
const element = $('selector1, selector2').find('img');
```

as it supports multiple selectors.

Reference: https://api.jquery.com/multiple-selector/